src/main/java/com/github/rodionovsasha/commentservice/services/UserService.java

```diff
@@ -0,0 +1,21 @@
+package com.github.rodionovsasha.commentservice.services;
+
+import com.github.rodionovsasha.commentservice.entities.User;
+
+public interface UserService {
+    User create(String name, int age);
+
+    User getById(long id);
```

Comment:
Can `getById` be a private class method(not an interface one)? I doubt any controller is going to use it